testnets/entrypointtestnet/chain.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "entrypointtestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "EntryPoint Testnet",
+  "chain_id": "entrypoint-pubtest-1",
+  "bech32_prefix": "entrypoint",
+  "daemon_name": "entrypointd",
+  "node_home": "$HOME/.entrypointd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uentry",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.0,
+        "average_gas_price": 0.0,
+        "high_gas_price": 0.0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uentry"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/entrypoint-zone/testnets",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": [
+      "v1.0.0"
+    ],
+    "cosmos_sdk_version": "0.47.4",
+    "consensus": {
```

Comment:
Can you please copy to consensus and sdk version into the first object in the 'version' array?

Reply:
Sorry I don't understand exactly @JeremyParish69. I already have consensus and sdk version in the 'versions' array.

I copied the format used in the Osmosis testnet:
- Consensus and sdk version under 'codebase': https://github.com/cosmos/chain-registry/blob/master/testnets/osmosistestnet/chain.json#L39-L42
- Consensus and sdk version under 'versions': https://github.com/cosmos/chain-registry/blob/master/testnets/osmosistestnet/chain.json#L56-L60